bugfix/fwf-4453:Fixed soringtby resetting sort values from sortmodal in all tables

fahad-aot opened the pull request:
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-4453
Issue Type: BUG

# Changes
1. Fixed sorting from sort modal

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

arun-s-aot commented:
The solution seems good. Can we use a reusable helper function in a util file to return the result ? This will always take the array as parameter and return the formatted result  and set to resetSortOrders variable in all the files ? @fahad-aot

shuhaib-aot reviewed (approved):
yes correct we can write a reset function in utils and use  where ever we want

fahad-aot commented:
> The solution seems good. Can we use a reusable helper function in a util file to return the result ? This will always take the array as parameter and return the formatted result and set to resetSortOrders variable in all the files ? @fahad-aot

Added helper function for resetting the sortorders in MF . Updated in this PR https://github.com/AOT-Technologies/forms-flow-ai-micro-front-ends/pull/537 . @arun-s-aot

arun-s-aot commented:
ignoring the sonar issues as it is coming in unit test files
